### PR TITLE
Index crawl

### PR DESCRIPTION
**Pull request by relud**
r? @oyiptong

### CODE REVIEW

**Comment by karlht** [comment]
r+ with nits in enclosed patch.

File: scripts/tile_index_crawl.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        help='Report SUCCESS',
+    )
+    options, args = parser.parse_args()
+
+    try:
+        from splice.environment import Environment
+        config = Environment.instance().config
+        cdn = config.CLOUDFRONT_BASE_URL
+        tile_index_key = config.S3['tile_index_key']
+    except Exception:
```
Comment (tspurway) [inline scripts/tile_index_crawl.py]: what is the expected exception here?  Shouldn't we be dumping a message to stdout?

Comment (relud) [inline scripts/tile_index_crawl.py]: the expected exception is that splice isn't in the module path, because the script is being run elsewhere.

**Pull request closed without merging**